tests/browser/recurring_donations/create_recurring_donation.robot

```diff
@@ -2,17 +2,15 @@
 
 Resource        tests/NPSP.robot
 Suite Setup     Open Test Browser
-Suite Teardown  Delete Records and Close Browser
+#Suite Teardown  Delete Records and Close Browser
```

Comment:
Why is this commented?

Reply:
was done while testing and forgot to update